README.md

```diff
@@ -35,7 +35,7 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-## Kanban board
+## Kanban board - test
```

Comment:
_note: Since this PR does not include all the content, I'll include my comments here._

## Frontend:

- [ ] Your frontend is beautiful.  Please, kindly include a screenshot in your readme to show it off. 💪 

- [ ] This text is superfluous because these are just instructions for you, not for your users:
![image](https://user-images.githubusercontent.com/57421823/208174854-e68dcc4e-9b2d-49ec-a892-6ab07bed9e70.png)
kindly remove it.


- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] I cannot reserve bikes.  When I try to reserve, this happens:

### Firefox
![image](https://user-images.githubusercontent.com/57421823/208175310-792f0089-89e5-4ede-8bab-de72f163d2ab.png)

### Chrome
![image](https://user-images.githubusercontent.com/57421823/208175531-fa6d398d-7496-4df3-8541-4fa20cb90229.png)


Kindly fix this bug so your app works as intended.  If this is not a bug, and your app requires special setup and/or permissions, kindly add them to your readme. 👍 

- [ ] It is not possible to know if I'm signed in or not, or who is the user that is signed in:

![image](https://user-images.githubusercontent.com/57421823/208175818-9e887ab2-02fd-4774-9198-59a4c87a7299.png)

Kindly add some way to confirm that a user is signed in, and who is it.

- [ ] In the "Add Reserve" button, the label is "Add new bike". 

![image](https://user-images.githubusercontent.com/57421823/208176074-207badf5-2edf-4ba6-a3e4-fdd701db1659.png)

This could lead to confusion because this seems to be the reservations form, not the add-bike form.  Kindly correct the header on the section to reflect the correct functionality of that section.

- [ ] The mobile view has many CSS overflows and misalignments:

![image](https://user-images.githubusercontent.com/57421823/208176272-a63dc63c-a7e0-400b-a8cc-4a1bc63d3f85.png)

![image](https://user-images.githubusercontent.com/57421823/208176531-86b4b2ea-37b5-481f-b9c0-740560aa5e61.png)

Kindly correct them, so your app shows on mobile devices as intended and looks more professional. 👍